### DISCUSSION TITLE
请问这个工程down下来yarn build一直失败

### DISCUSSION THREAD

**Discussion by wangjie19910615** [alibaba/designable#28] [Q&A]
yarn run v1.22.5
$ dumi dev
Command failed: D:\A develop_project\develop\new\designable\node_modules\.bin\lerna ls --json --all
'D:\A' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���

Error: Command failed: D:\A develop_project\develop\new\designable\node_modules\.bin\lerna ls --json --all
'D:\A' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���

    at checkExecSyncError (child_process.js:643:11)

**Comment by dahui4dev** [comment]
windows 本，文件夹路径不要有空格。莫名其妙的问题坑死你。

**Reply by wangjie19910615** [reply]
还真是

**Reply by dahui4dev** [reply]
软件开发第一原则：path 不要有空格，不要有中文

**Reply by wangjie19910615** [reply]
感谢，用windows用习惯了，还有个问题想请教下，这个designable的配置响应器好像没有不在

**Reply by wangjie19910615** [reply]
和官网的https://designable-antd.formilyjs.org/有很多不同，配置的不是jsonSchema

**Reply by dahui4dev** [reply]
你是不是找这个代码？ 编辑 formily 的？https://github.com/alibaba/formily/tree/formily_next/designable/antd

**Reply by wangjie19910615** [reply]
是的，是这个

**Reply by wangjie19910615** [reply]
我想用这个编辑 formily做为一个独立的工程
